Makefile

```diff
@@ -5,4 +5,13 @@
 # https://www.viget.com/articles/two-ways-to-share-git-hooks-with-your-team/
 .PHONY: init
 init:
-	git config core.hooksPath .githooks
\ No newline at end of file
+	git config core.hooksPath .githooks
+
+.PHONY: install-go-tools
+install-go-tools:
+	go install golang.org/x/tools/cmd/goimports
+
+.PHONY: go-fmt
+go-fmt: install-go-tools
+	gofmt -w .
+	goimports -w .
```

Comment:
doesn't goimports [also run gofmt](https://pkg.go.dev/golang.org/x/tools/cmd/goimports?utm_source=godoc)?
i think we can remove the gofmt line